### PR TITLE
Installing Python 3.6 in Ubuntu

### PR DESCRIPTION
**Pull request by kaapstorm**
@dimagi/py3 

@nickpell @esoergel buddy @czue

### CODE REVIEW

**Review by nickpell — APPROVED** [review]

**Review by emord — COMMENTED** [review]
@nickpell I think that staging is now on ubuntu 18.04 which does have 3.6. I don't see anything in commcare-cloud including a PPA, so does this mean we will block py3 on upgrading to bionic?

**Review by emord — APPROVED** [review]

File: DEV_SETUP.md [hunk 1]
```diff
@@ -73,6 +73,11 @@ Note that once you're up and running, you'll want to periodically re-run these s
 
 - Install [Python 3.6](https://www.python.org/downloads/)
     - For OSX, you can [install using Homebrew](http://osxdaily.com/2018/06/13/how-install-update-python-3x-mac/)
+    - For Ubuntu, you will need both the `python3.6` and `python3.6-dev`
+      packages. If your version of Ubuntu doesn't come with Python 3.6
+      pre-installed or in its standard repos, you can use the
+      [deadsnakes PPA](https://launchpad.net/~deadsnakes) to install version
```
[the remark this answers is not in the document]
Comment (esoergel) [reply DEV_SETUP.md]: nothing inspires trust like good ol' `deadsnakes`

[the remark this answers is not in the document]
Comment (emord) [reply DEV_SETUP.md]: seems like a strange name given that they're providing more recently alive pythons

[the remark this answers is not in the document]
Comment (nickpell) [reply DEV_SETUP.md]: 🐍 🎶

**Pull request merged**